cmd/nerdctl/container_stats.go

```diff
@@ -329,7 +329,7 @@ func statsAction(cmd *cobra.Command, args []string) error {
 					if err := tmpl.Execute(&b, rc); err != nil {
 						break
 					}
-					if _, err = fmt.Fprintf(cmd.OutOrStdout(), b.String()+"\n"); err != nil {
+					if _, err = fmt.Fprint(cmd.OutOrStdout(), b.String()+"\n"); err != nil {
```

Comment:
`fmt.Fprintln(cmd.OutOrStdout(), b.String())`

Reply:
Make sense.

Reply:
Can we merge this PR, or do you plan to update this PR?

Reply:
Plan to update. Traveling.